README.md

```diff
@@ -56,19 +57,50 @@ npm run build-webapp
 Follow these instructions to run FINALE:
 - Once you have the repository built and set up according to the above commands, proceed to the next step.
 
-- ### Converting LabVIEW code to JSON: 
-   - Navigate to "buid/HTMLGenerator/".
-
-   - Open Main.vi and enter values for the following:
-     - Source directory/files: Path to the source LabVIEW code file(s) or folder.
-     - Destination Directory: Path to the destination directory. To view the files using the WebApp, make sure your destination is set to "<Path/to/FINALE/repo>/build/src".
-     - Run Main.vi and click "Convert".
-   ![Main.vi](./docs/Main.vi.png)
-     > Note: If you require certain files/projects to be preloaded for converting  the files, open GeneratorUI-Advanced.vi and enter values for the following:
-     >  - Top level output path: <Path/to/FINALE/repo>/build/src
-     >  - Files to Preload: Array of files you want to preload. If this is  left empty it is equivalent to running Main.vi.
-     >  - File(s)/Folder to convert: Path to the source LabVIEW code file(s) or  folders.
-     >  - Destination Folder (relative to output path): This is an optional  field to specify an output path for the converted files. This must  be relative to the Top level output path.
+- ### Converting LabVIEW code to FINALE format
+  This can be done in two ways:
+
+    - #### Running the Converter VI: 
+      - Navigate to "buid/HTMLGenerator/".
+
+      - Open Main.vi and enter values for the following:
+        - Source directory/files: Path to the source LabVIEW code file(s) or folder.
+        - Destination Directory: Path to the destination directory. To view the files using the WebApp, make sure your destination is set to "<Path/to/FINALE/repo>/build/src".
+        - Run Main.vi and click "Convert".
+      ![Main.vi](./docs/Main.vi.png)
+        > Note: If you require certain files/projects to be preloaded for converting  the files, open GeneratorUI-Advanced.vi and enter values for the following:
+        >  - Top level output path: <Path/to/FINALE/repo>/build/src
+        >  - Files to Preload: Array of files you want to preload. If this is  left empty it is equivalent to running Main.vi.
+        >  - File(s)/Folder to convert: Path to the source LabVIEW code file(s) or  folders.
+        >  - Destination Folder (relative to output path): This is an optional  field to specify an output path for the converted files. This must  be relative to the Top level output path.
+
+    - #### Using CLI:
+      - On cmd or powershell, navigate to <Path/of/FINALE/repo>/build/HTMLGenerator.
```

Comment:
Minor: Can you go through the document to fix the capitalizations? It should be capitalized PowerShell instead of powershell, for example.